### ISSUE TITLE
Add `affected rows` on mutation response

### ISSUE THREAD

**Issue by bossjapzz** [supabase/pg_graphql#90]
**Describe the bug**
Add `affected rows` on mutation response

**To Reproduce**
1. Create/Update/Delete any records from table

**Expected behavior**
1. Must return the affected rows from the mutation

**Screenshots**
<img width="326" alt="Screen Shot 2022-02-21 at 8 31 44 PM" src="https://user-images.githubusercontent.com/100037148/154956065-4b3d4b4a-8210-44dd-b13a-621467f4b021.png">

**Request**
```graphql
mutation {
  insert_Group(objects: {id: "d", name: "d", shortname: "d", description: "d", updatedAt: "2022-02-21"}) {
    affected_rows
    returning {
      id
      name
      shortname
    }
  }
}

```

**Response**
```json
{
  "data": {
    "insert_Group": {
      "affected_rows": 1,
      "returning": [
        {
          "id": "WzEsICJwdWJsaWMiLCAiR3JvdXAiLCAiZCJd",
          "name": "d",
          "shortname": "d"
        }
      ]
    }
  }
}
```

**Comment by olirice** [comment]
The return types for update and delete mutations are

```graphql
type AccountDeleteResponse {
  """Count of the records impacted by the mutation"""
  affectedCount: Int!

  """Array of records impacted by the mutation"""
  records: [Account!]!
}

type AccountUpdateResponse {
  """Count of the records impacted by the mutation"""
  affectedCount: Int!

  """Array of records impacted by the mutation"""
  records: [Account!]!
}
```

Comparing to the hasura schema, `affected_row` = `affectedCount` and `returning` = `records`.

`pg_graphql` does not currently have a bulk insert mutation, but when it does, it will follow the same structure

**Comment by olirice** [comment]
resolved by 43f4bfa98a57fae82a682141d3dae7d454ed89c1

https://supabase.github.io/pg_graphql/reflection/
